### PR TITLE
[SINT-5371] Updated Python SDK v1 syntax and runtime to Python3.9+

### PR DESCRIPTION
**Pull request by marcos-vivar**
## TL;DR

- Deprecated TS Python SDK v1 support for Python 2
- Deprecated TS Python SDK v1 support for Python 3.7
- Updated TS Python SDK v1 support for Python3.9
- Updated TS Python SDK v1 support for Python3.10

## What?
Several projects are depending on TS Legacy SDKs and they needed to be upgraded so the systems they support can be safely updated as well.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -174,7 +174,4 @@ dmypy.json
 # Ignore all local history of files
 .history
 
-
-trustar.conf*
```
Comment (dsolmirano) [inline .gitignore:178]: keep this line

**Review by dsolmirano — APPROVED** [review]

**Review by dsolmirano — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -53,14 +53,8 @@ To setup this project for development, you need to have [pipenv](https://pipenv.
...
-pipenv run pip freeze > requirements.txt
-```
-
-Note: you can use pipenv too to create a 2.7 venv, check the pipenv docs for that.
-
-### Python 2/3 Compatibility
-
-This package is compatible with both Python 2 and 3.  Ensure that any changes made maintain this cross-compatibility.
+### Python 3 Compatibility
+This package is no longer compatible with `Python 3.7`
```
Comment (zamboriandras) [inline README.md:60]: We could just reword this to say minimum version for compatibility is Python 3.9

**Review by zamboriandras — APPROVED** [review]
Just one small comment in the Readme

File: README.md (hunk 2 above)
Comment (marcos-vivar) [reply README.md]: Addressed, and also included the `tox` setup intructions 👍🏽 

Thanks for the feedback 🥇
